ui/src/authorizations/components/BucketsTokenOverlay.tsx

```diff
@@ -175,11 +203,21 @@ class BucketsTokenOverlay extends PureComponent<Props, State> {
       params: {orgID},
       onCreateAuthorization,
     } = this.props
+    const {activeTabRead, activeTabWrite} = this.state
 
-    const permissions = [
-      ...this.writeBucketPermissions,
-      ...this.readBucketPermissions,
-    ]
+    let permissions = []
+
+    if (activeTabRead === BucketTab.Scoped) {
+      permissions = [...permissions, ...this.readBucketPermissions]
```

Comment:
nit: this could just be `permissions = [...this.readBucketPermissions]` here.

Reply:
Yup, good idea :)